extensions/wikia/Search/classes/Test/HooksTest.php

```diff
@@ -86,6 +86,7 @@ public function testOnArticleDeleteComplete() {
 		    ->will   ( $this->returnValue( true ) )
 		;
 		$this->mockClass( 'Wikia\Search\Indexer', $mockIndexer );
+		$this->mockStaticMethod( 'Wikia\Search\Indexer', 'canIndex', true );
 		$this->assertTrue(
 				$mockHooks->onArticleDeleteComplete( $mockArticle, $mockUser, 'why not', 123 )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "why not" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)